### DISCUSSION TITLE
Is it possible to set up a custom container name when running tasks on ECS?

### DISCUSSION THREAD

**Discussion by biancarosa** [dagster-io/dagster#18848] [Q&A]
I'd like to monitor my tasks CPU/Memory consumption, and the container name is how I was thinking of grouping them - since they all run with the same image, task definition, etc etc.

My idea was to spin up the container with the job name instead of spinning it up with the same container name as the job. Looking at boto3's run task api, I don't see a way of overriding the container name, but I don't know if I'm missing something here.

**Comment by prha** [comment]
@biancarosa We do allow you to specify the container name in the run launcher config, but it is static (see [docs](https://docs.dagster.io/_apidocs/libraries/dagster-aws#ecs)).  I take it that you're looking to vary the container name on a job-to-job basis?

**Reply by biancarosa** [reply]
Yes! I'd like each container to have a name that contains the job name.
